rpyscope/gui_qt.py

```diff
@@ -36,7 +37,16 @@ class MainWindowControls(QMainWindow):
     def __init__(self):
         # info variables
         self.version = "0.0.1"
-        self.author = "Reto Trappitsch"
```

Comment:
This should also be edited in `setup.py`. Alternatively - and maybe better - we could kick it out of here (since I don't think it is used anywhere) and create an `AUTHORS.md` file in the repo. I don't have strong feelings about this (at this point) either way.

Reply:
Ok I'll go for `AUTHORS.md`